full-parent/pom.xml

```diff
@@ -15,21 +15,16 @@
     <description>SmallRye Parent POM</description>
 
     <properties>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.testTarget>${maven.compiler.target}</maven.compiler.testTarget>
-        <maven.compiler.testSource>${maven.compiler.source}</maven.compiler.testSource>
-
         <!-- Dependency versions -->
-        <version.jakarta.api>8.0.0</version.jakarta.api>
-        <version.jakarta.annotation.api>1.3.5</version.jakarta.annotation.api>
-        <version.jakarta.enterprise.cdi.api>2.0.2</version.jakarta.enterprise.cdi.api>
-        <version.jakarta.json-p.api>1.1.6</version.jakarta.json-p.api>
-        <version.jakarta.json-b.api>1.0.2</version.jakarta.json-b.api>
-        <version.jakarta.ws.rs.api>2.1.6</version.jakarta.ws.rs.api>
-        <version.jakarta.jms.api>2.0.3</version.jakarta.jms.api>
+        <version.jakarta.api>9.1.0</version.jakarta.api>
+        <version.jakarta.annotation.api>2.0.0</version.jakarta.annotation.api>
+        <version.jakarta.enterprise.cdi.api>3.0.0</version.jakarta.enterprise.cdi.api>
+        <version.jakarta.json-p.api>2.0.1</version.jakarta.json-p.api>
+        <version.jakarta.json-b.api>2.0.0</version.jakarta.json-b.api>
+        <version.jakarta.ws.rs.api>3.0.0</version.jakarta.ws.rs.api>
+        <version.jakarta.jms.api>3.0.0</version.jakarta.jms.api>
```

Comment:
We probably want the Jakarta EE 10 dependencies here?

Reply:
Yes, we could update them too. Sec.

Reply:
Ok, I made some tests in some downstream projects to ensure this was ok.